xcell/mappers/mapper_CatWISE.py

```diff
@@ -0,0 +1,117 @@
+from .mapper_base import MapperBase
+from .utils import get_map_from_points
+from astropy.table import Table
+import numpy as np
+import healpy as hp
+
+
+class MapperCatWISE(MapperBase):
+    def __init__(self, config):
+        """
+        config - dict
+          {'data_catalog': 'xcell/tests/data/
+          catwise_agns_masked_final_w1lt16p5_alpha.fits',
+           'mask': 'xcell/tests/data/MASKS_exclude_master_final.fits',
+           'mask_name': 'mask_CatWISE'}
+        """
+        self._get_defaults(config)
+        self.file_sourcemask = config.get('mask_sources', None)
+        self.ra_name = 'ra'
+        self.dec_name = 'dec'
```

Comment:
do we need to store these two (self.ra_name, self.dec_name) for any reason?

Reply:
actually we don't